meta-mender-core/recipes-mender/mender-connect/mender-connect.inc

```diff
@@ -53,21 +48,14 @@ do_prepare_mender_connect_conf[vardeps] = " \
     MENDER_CONNECT_USER \
 "
 
-do_install() {
-    oe_runmake \
-        -C ${B}/src/${GO_IMPORT} \
-        V=1 \
-        prefix=${D} \
-        bindir=${bindir} \
-        datadir=${datadir} \
-        sysconfdir=${sysconfdir} \
-        systemd_unitdir=${systemd_unitdir} \
-        install-bin \
-        install-systemd
-
-    # install configuration
-    mkdir -p  ${D}/${sysconfdir}/mender
+do_install:append() {
+    # Install configuration
+    mkdir -p ${D}/${sysconfdir}/mender
     install -m 0600 ${B}/mender-connect.conf ${D}/${sysconfdir}/mender/mender-connect.conf
+
+    # Install systemd service
+    install -m 755 -d ${D}${systemd_unitdir}/system
+    install -m 0644 ${B}/src/${GO_IMPORT}/support/mender-connect.service ${D}${systemd_unitdir}/system/
```

Comment:
Pre-existing,

I noticed that the install of the systemd service for this recipe is not guarded with the `mender-systemd` feature. This was wrong and needs fixing (separate commit with changelog). It can look like:

```
do_install:append() {
    # Install configuration
    mkdir -p ${D}/${sysconfdir}/mender
    install -m 0600 ${B}/mender-connect.conf ${D}/${sysconfdir}/mender/mender-connect.conf
}

do_install:append:mender-systemd() {
    # Install systemd service
    install -m 755 -d ${D}${systemd_unitdir}/system
    install -m 0644 ${B}/src/${GO_IMPORT}/support/mender-connect.service ${D}${systemd_unitdir}/system/
}
```